docker-compose.yml

```diff
@@ -0,0 +1,92 @@
+version: "2.1"
+services:
+  eureka:
+   image: aypas/eureka-server
```

Comment:
indentation should be consistent in this file. 1-space is unconventional but i won't force you to use 2-space.